pkg/cmd/get.go

```diff
@@ -90,6 +94,27 @@ func (o *GetOptions) Complete(args []string) error {
 
 	o.RESTConfig = config
 
+	if o.Output != "" {
+		err = o.Validate()
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (o *GetOptions) Validate() error {
+	validOutputFormats := []string{"json", "yaml"}
+	o.IsValidOutput = false
+	for _, validOutputFormat := range validOutputFormats {
+		if validOutputFormat == o.Output {
+			o.IsValidOutput = true
+		}
+	}
+	if !o.IsValidOutput {
+		return fmt.Errorf("unable to match a printer suitable for the output format %s, allowed formats are: json, yaml", o.Output)
```

Comment:
```suggestion
		return fmt.Errorf("unable to match a printer suitable for the output format %s, allowed formats are: %s", o.Output, strings.Join(validOutputFormats, ", "))
```

Reply:
that makes sense since I defined validOutputFormats :) Thanks!

Reply:
Changed